org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/chrome/ChromeRunDAPDebugDelegate.java

```diff
@@ -11,98 +11,109 @@
 		if (!argsString.isEmpty()) {
 			Object[] args = Arrays.asList(argsString.split(" ")).stream() //$NON-NLS-1$
 					.filter(s -> !s.trim().isEmpty()).toArray();
 			if (args.length > 0) {
-				param.put(ARGUMENTS, args);
+				param.put(AbstractDebugDelegate.ARGUMENTS, args);
 			}
 		}
-		Map<String, String> env = configuration.getAttribute(ILaunchManager.ATTR_ENVIRONMENT_VARIABLES,
+
+		// Debug environment variables
+		Map<String, String> env = configuration.getAttribute(AbstractDebugDelegate.ENV,
 				Collections.emptyMap());
 		if (!env.isEmpty()) {
 			JsonObject envJson = new JsonObject();
 			env.forEach((key, value) -> envJson.addProperty(key, value));
-			param.put(ENV, envJson);
+			param.put(AbstractDebugDelegate.ENV, envJson);
 		}
-		String cwd = configuration.getAttribute(DebugPlugin.ATTR_WORKING_DIRECTORY, "").trim(); //$NON-NLS-1$
+		
+		// Chrome working directory
+		String cwd = configuration.getAttribute(AbstractDebugDelegate.CWD, "").trim(); //$NON-NLS-1$
 		if (!cwd.isEmpty()) {
-			param.put(CWD, cwd);
+			param.put(AbstractDebugDelegate.CWD, cwd);
 		}
 		// workaround until
 		// https://github.com/microsoft/vscode-node-debug2/commit/f2dfa4ca4026fb3e4f143a391270a03df8187b42#diff-d03a74f75ec189cbc7dd3d2e105fc9c9R625
 		// is released in VSCode
-		param.put("sourceMaps", false);
-		param.put("runtimeExecutable", findChromeLocation());
-		param.put("file", configuration.getAttribute(PROGRAM, "no program path defined"));
-		param.put("trace", "verbose");
+		param.put(AbstractDebugDelegate.SOURCE_MAPS, false);
+		
+		// TODO: Let user point to the location of their Chrome executable
+		param.put(RUNTIME_EXECUTABLE, findChromeLocation());
+		
+		if (configuration.getAttribute(VERBOSE, false)) {
+			param.put(TRACE, VERBOSE);
+		}
 
+		super.launchWithParameters(configuration, mode, launch, monitor, param, findDebugAdapter());
+	}
+	
+	static File findDebugAdapter() {
+		URL fileURL;
 		try {
-			URL fileURL = FileLocator.toFileURL(
-					getClass().getResource("/language-servers/node_modules/debugger-for-chrome/out/src/chromeDebug.js"));
-			//"/language-servers/node_modules/vscode-firefox-debug/out/adapter/firefoxDebugAdapter.js"
-			File file = new File(fileURL.getPath());
-			List<String> debugCmdArgs = Collections.singletonList(file.getAbsolutePath());
-
-			DSPLaunchDelegateLaunchBuilder builder = new DSPLaunchDelegateLaunchBuilder(configuration, mode, launch,
-					monitor);
-			builder.setLaunchDebugAdapter(InitializeLaunchConfigurations.getNodeJsLocation(), debugCmdArgs);
-			builder.setMonitorDebugAdapter(configuration.getAttribute(DSPPlugin.ATTR_DSP_MONITOR_DEBUG_ADAPTER, false));
-			builder.setDspParameters(param);
-
-			super.launch(builder);
-		} catch (IOException e) {
+			// TODO: Change location of Chrome Debug Adapter to node_modules folder
+			fileURL = FileLocator.toFileURL(
+					ChromeRunDAPDebugDelegate.class.getResource("/language-servers/chrome-debug-adapter/package/out/src/chromeDebug.js"));
```

Comment:
the debuger location should be `"/language-servers/node_modules/debugger-for-chrome/out/src/chromeDebug.js"`